Task_TheGridProblem/index.css

```diff
@@ -1 +1,51 @@
-/* Start coding below for GRID PROBLEM TASK */
+/* Start coding below for GRID PROBLEM TASK */
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.container{
+    gap: 3px;
+    display: grid;
+    position: relative;
+    grid-template-columns: repeat(6,1fr);
+    grid-template-rows: repeat(6,1fr);
+    height: 700px;
+    width: 700px;
+    color: aliceblue;
+}
+
+.one{
+    background-color: #640505;
+    grid-column: 1 / 3;
+    grid-row: 1 / 3;
+}
+.two{
+    background-color: #640505;
+    grid-column: 3 / 5;
+    grid-row: 1 / 3;
+}
+.three{
+    background-color: #640505;
+    grid-column: 5 / 7;
+    grid-row: 1 / 3;
+}
+
+.four{
+    background-color: #FF0000;
+    grid-column: 1 / 4;

```

Comment:
The grid-row value in class 'four' is not correctly specified. The end row should come before the start row for proper grid placement.